src/dialogs/more-info/controls/more-info-climate.js

```diff
@@ -309,11 +309,11 @@ class MoreInfoClimate extends EventsMixin(PolymerElement) {
   }
 
   supportsTemperature(stateObj) {
-    return (stateObj.attributes.supported_features & 1) !== 0;
+    return ((stateObj.attributes.supported_features & 1) !== 0) && stateObj.attributes.temperature;
```

Comment:
temperature can be 0 (= false), some for other attributes

Reply:
That is what you get for reusing previous code :-) I will fix it by using @balloob suggestion